src/v3/components/loading/loading.styles.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box } from '@mui/material';
+import { styled } from '@mui/system';
+
+export const StyledCircleBox = styled(Box)({
+    height: '156px',
+    width: '156px',
+    backgroundColor: 'var(--always-white-base)',
+    borderRadius: 'var(--radius-xxl, 16px)',
+    display: 'flex',
+    padding: '24px 40px',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    boxShadow: '0px 16px 24px 0px rgba(28, 28, 28, 0.16)',
```

Comment:
@yrakhwas: would it be possible to use the `pop-up-shadow` class name to define this shadow effect? Thank you.